src/terminal/color_scheme.c

```diff
@@ -19,7 +19,7 @@
 
 #include "config.h"
 
-#include "terminal/color-scheme.h"
+#include "terminal/color_scheme.h"
```

Comment:
I renamed these files to match the underscore convention that most of the rest of this library is following.

Reply:
Hm ... I think these should be renamed back. If the rest of the library is following an underscore convention for filenames, I'd lean toward migrating that to dashes as part of public-ifying the library. The dash convention is the convention already used by libguac for its public headers.

Reply:
Oh, sure. can do.